index.html

```diff
@@ -2683,10 +2480,10 @@ <h4 id="dependency-management">Success Criterion: Dependency management</h4>
 					<p class="external"><a class="machine" href="https://w3c.github.io/sustainableweb-wsg/star.html#WD16-1">Machine-testable</a> <span class="hide">and </span><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#WD16-1">Resources</a></p>
 					<p>Prevent developers from downloading and installing libraries and frameworks to run client-side when they are not needed by checking for unused dependencies. Follow up by uninstalling those that are not needed.</p>
 				</section>
-				<section class="notoc" data-testable="machine" data-materials="low" data-energy="low" data-water="low" data-emissions="low" data-standard="GPF, GR491, SDGs" data-considerations="Security" data-categories="JavaScript, Patterns, Performance, Software">
+				<section class="notoc" data-testable="machine" data-materials="low" data-energy="low" data-water="low" data-emissions="low" data-standard="GPF, GR491, SDGs" data-considerations="Privacy, Security" data-categories="JavaScript, Patterns, Performance, Software">
 					<h4 id="dependency-necessity">Success Criterion: Dependency necessity</h4>
 					<p class="external"><a class="machine" href="https://w3c.github.io/sustainableweb-wsg/star.html#WD16-2">Machine-testable</a> <span class="hide">and </span><a class="urls" href="https://w3c.github.io/sustainableweb-wsg/resources.html#WD16-2">Resources</a></p>
-					<p>Limit your use of libraries and frameworks to the genuinely necessary as this will reduce the amount of code that has to be downloaded and parsed by the browser. Consider whether you can use vanilla code instead. Check the package size and whether individual modules can be installed and imported individually, as opposed to the entire library.</p>
+					<p>Limit your use of libraries and frameworks to the genuinely necessary as this will reduce the amount of code that has to be downloaded and parsed by the browser. Consider whether you can use vanilla code instead. Check the package size and whether individual modules can be installed and imported individually or a more performant alternative can be used in its place, as opposed to the entire library.</p>
```

Comment:
I recall discussing the term "vanilla code" as an idiom that might not be understood. Can we consider changing it to "plain code" or "code without extra libraries"?

Reply:
we could also consider removing “Consider whether you can use vanilla code instead.” altogether, as the use of plain/vanilla is implied by the previous sentence.

And, that way, we can avoid debate on whether libraries and frameworks _are_ indeed vanilla 'under the hood'. (I've been there :D )